sim/warrior/deep_wounds.go

```diff
@@ -66,16 +66,15 @@ func (warrior *Warrior) applyDeepWounds() {
 	}))
 }
 
+// Needs confirmation on classic functionality.  It is regular dot instead of a stacking one and if so does that mean it is only overwritten by a "higher" crit?
 func (warrior *Warrior) procDeepWounds(sim *core.Simulation, target *core.Unit, isOh bool) {
 	dot := warrior.DeepWounds.Dot(target)
 
-	outstandingDamage := core.TernaryFloat64(dot.IsActive(), dot.SnapshotBaseDamage*float64(dot.NumberOfTicks-dot.TickCount), 0)
-
 	var awd float64
 	if isOh {
 		attackTableOh := warrior.AttackTables[target.UnitIndex][proto.CastType_CastTypeOffHand]
 		adm := warrior.AutoAttacks.OHAuto().AttackerDamageMultiplier(attackTableOh)
-		awd = warrior.AutoAttacks.OH().CalculateAverageWeaponDamage(dot.Spell.MeleeAttackPower()) * 0.5 * adm
+		awd = warrior.AutoAttacks.OH().CalculateAverageWeaponDamage(dot.Spell.MeleeAttackPower()) * 0.5 * adm // Does this not include the DWS talent?
```

Comment:
Deep wound is affected and double dip from damage modifier in Classic.

Reply:
Does the * .5 need to have dual weapon spec modifying it as well (assuming its *0.5 because of offhand damage?)

Reply:
Correct. It is halved because it is from offhand. However, that is the base damage for the dot calculation. The dot damage itself doesn't get halved but benefit from dual wield specialization. Deep wound double dipping was removed at SoD at one point. So it'd have to be reimplemented.

Reply:
Thanks I'll dig through the old commits and see

Reply:
Ok updated with the old double dip code.